ci/ci/ci.py

```diff
@@ -45,6 +45,10 @@ def status():
 @app.route('/push', methods=['POST'])
 def github_push():
     d = request.json
+    if 'zen' in d:
```

Comment:
is there an api doc you can point me to?

Reply:
I cannot. I have deduced that GitHub sends "zen" when you register a web hook. That was causing 500s because the document was missing all the expected fields of a `push` event.

Reply:
I literally have no idea where this is documented.

Reply:
ha, ok

Reply:
It is here, search for zen: https://developer.github.com/webhooks/

Reply:
Your google-fu is impressive.